src/client/components/Loader/index.js

```diff
@@ -1,5 +1,199 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-export default function Loader() {
-  return <div>Loading...</div>;
+export default function Loader({ scale, color }) {
+  return (
+    <svg
```

Comment:
Hey. I can see that this component workes the same as the spinner. Probably it will be better to have only one in the end.

Reply:
Yeah, good that you say. Left it there for a reason :D To remind myself.
`import { Home } from './containers/Home/Home';
import SignIn from './containers/SignIn';
import SignUp from './containers/SignUp';
import ResetPassword from './containers/ResetPassword';
import AuthenticatedRoute from './components/Auth/AuthenticatedRoute';
import { useAuthentication } from './hooks/useAuthentication';
import { Header } from './components/Header/Header';
import Profile from './containers/Profile';
import Loader from './components/Loader';
import Menu from './components/Menu/Menu';
import Footer from './components/Footer/Footer';`

What type of import is the accepted ? default or named ?

Reply:
You may test it now.
Insert <Loader /> in App.js fx. before <Header />

Reply:
I can't find a rule for that. But probably the default one it's going to be the most used one, often vsc does it for you.